src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilterMutectCalls.java

```diff
@@ -143,7 +145,19 @@ public void closeTool() {
         }
     }
 
-    private String getTumorSampleName(){
-        return getHeaderForVariants().getMetaDataLine(Mutect2Engine.TUMOR_SAMPLE_KEY_IN_VCF_HEADER).getValue();
+    private String getTumorSampleName() {
+        if (!MTFAC.mitochondria) {
```

Comment:
This can be a one-liner: `return MTFAC.mitochondria ? getMitoSampleName() : getHeaderForVariants().getMetaDataLine(Mutect2Engine.TUMOR_SAMPLE_KEY_IN_VCF_HEADER).getValue();`